mkdocs.yml

```diff
@@ -0,0 +1,82 @@
+site_name: Claudie Docs
+site_url: https://docs.claudie.io/
+repo_url: https://github.com/berops/claudie
+repo_name: claudie
+
+markdown_extensions:
+  - admonition
+  - pymdownx.details
+  - pymdownx.superfences
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.tasklist:
+      custom_checkbox: true
+  - toc:
+      permalink: true
+
+plugins:
+  - mike:
+      canonical_version: null
+      version_selector: true
+  - search:
+      lang: en
+  - glightbox:
+      width: 100%
+      height: auto
+      zoomable: true
+
+copyright: Copyright &copy; 2023 BerOps s.r.o.
+
+theme:
+  name: material
+  features:
+    - content.code.copy
+    - navigation.top
+  palette:
+    - primary: white
+    - accent: black
+  logo: hexagon_blue.png
+  favicon: hexagon_blue.png
+
+nav:
+  - index.md
+  - Getting Started:
+    - getting-started/get-started-using-claudie.md
+    - CRUD for Claudie: crud/crud.md
+  - Input manifest: 
+      - input-manifest/input-manifest.md
+      - Example yaml file: input-manifest/example.md
+      - Providers:
+        - Examples:
+          - AWS example: input-manifest/providers/examples/aws-input-manifest.md
+          - Azure example: input-manifest/providers/examples/azure-input-manifest.md
+          - GPC example: input-manifest/providers/examples/gcp-input-manifest.md
+          - Hetzner example: input-manifest/providers/examples/hetzner-input-manifest.md
+          - OCI example: input-manifest/providers/examples/oci-input-manifest.md
+        - AWS: input-manifest/providers/aws.md
+        - Azure: input-manifest/providers/azure.md
+        - GCP: input-manifest/providers/gcp.md
+        - Cloudflare: input-manifest/providers/cloudflare.md
+        - Hetzner: input-manifest/providers/hetzner.md
+        - OCI: input-manifest/providers/oci.md
+  - How Claudie works:
+    - Claudie Workflow: claudie-workflow/claudie-workflow.md
+    - Claudie Storage solution: storage/storage-solution.md
+    - Loadbalancing in Claudie: ./loadbalancing/loadbalancing-solution.md
+    - Autoscaling in Claudie: autoscaling/autoscaling.md
+  - Claudie Use Cases: use-cases/use-cases.md
+  - Roadmap for Claudie: roadmap/roadmap.md
+  - Contributing:
+    - contributing/contributing.md
+    - Release a new version: contributing/release.md
+  - CHANGELOG:
```

Comment:
Two questions. 
1. Will the release pipeline append a new changelog if we release a new version, i.e. `v0.4.0`?
2. Should we keep it capitalized? (`Changelog` vs `CHANGELOG`)

Reply:
1. No it won't. Guess I will have to figure out the way, how to do that...
2. I just kept it capitalized, because the folder with the changelog markdown files is capitalized. So what's your opinion?

Reply:
I think I prefer `Changelog` on pretty docs, and `CHANGELOG` on Github. If that would be possible to do, I think that would be great. If not, I am okay with either.

Reply:
Done and deploy and already deployed on [docs.claudie.io](docs.claudie.io)